dask/dataframe/accessor.py

```diff
@@ -129,6 +129,7 @@ def split(self, pat=None, n=-1, expand=False):
                 delimiter = " " if pat is None else pat
                 meta = type(self._series._meta)([delimiter.join(["a"] * (n + 1))])
                 meta = meta.str.split(n=n, expand=expand, pat=pat)
+                meta = meta.iloc[:0].set_index(self._series._meta.index)
```

Comment:
This can be removed if you apply my suggestion from above. It's not needed now that we create a `meta` with the proper index.